server/index.js

```diff
@@ -77,6 +78,7 @@ if (isGAEProd) {
   handlers.unshift(compression());
 }
 
+app.get('/**/preview', previewHandler);
```

Comment:
Specific reason you did not add it to `handlers` as well?

Reply:
The other handlers are more of a middleware for serving the static site, rather than dynamic content, so I'd like to have that distiction.